requirements/requirements.in

```diff
@@ -11,7 +11,7 @@ newrelic
 oauthlib
 psycopg2
 pycryptodomex
-PyJWT
+PyJWT[crypto]
```

Comment:
This is the change in the requirements but rebuilding the deps updated quite a few versions.